python/ray/experimental/channel/torch_tensor_nccl_channel.py

```diff
@@ -163,6 +211,9 @@ def write(self, value: Any, timeout: Optional[float] = None) -> None:
         (3) on the first `write` call. The reader is expected to reuse the sent
         data for subsequent messages.
         """
+        if self._local_channel is not None:
+            self._local_channel.write(value)
```

Comment:
Can you update docstring regarding this part?

Reply:
update: https://github.com/ray-project/ray/pull/49213/commits/b4c7e2641102167a7df166c04e7d82c203099aa2